### PR TITLE
predict_confidence

### PR DESCRIPTION
**Pull request by mkborregaard**
Handle prediction that returns a confidence interval
cc @nalimilan
cf https://github.com/JuliaStats/GLM.jl/pull/253

### CODE REVIEW

File: src/statsmodel.jl [hunk 1]
```diff
@@ -48,12 +48,12 @@ end
...
-To specify that a model type `T` drops the intercept, overload this function for the 
+Define whether a given model automatically drops the intercept. Return `false` by default.
+To specify that a model type `T` drops the intercept, overload this function for the
 corresponding type: `drop_intercept(::Type{T}) = true`
 
-Models that drop the intercept will be fitted without one: the intercept term will be 
-removed even if explicitly provided by the user. Categorical variables will be expanded 
+Models that drop the intercept will be fitted without one: the intercept term will be
+removed even if explicitly provided by the user. Categorical variables will be expanded
 in the rank-reduced form (contrasts for `n` levels will only produce `n-1` columns).
```
Comment (mkborregaard) [inline src/statsmodel.jl]: Everything above here is just my text editor removing trailing whitespace

**Comment by mkborregaard** [comment]
Is this ready to go in, or do you need anything more from me?

**Comment by nalimilan** [comment]
That's fine with me (I've approved!).

**Comment by mkborregaard** [comment]
Thanks @nalimilan . @andreasnoack just remarked on Slack that he thinks the return type here should be a DataFrame not a 3-column Matrix. Do you agree with this? I can make a new PR.

**Comment by andreasnoack** [comment]
Maybe I didn't read the conversation carefully enough but it did seem to me as if people thought a table was appropriate here.

**Comment by mkborregaard** [comment]
This was merged before that discussion took place

**Comment by andreasnoack** [comment]
@mkborregaard Would you mind that this is reverted while you prepare the new PR? I'd like to get the GLM docs set up again but that requires a new release of StatsModels.

**Comment by mkborregaard** [comment]
That's fine I expect to fix this tomorrow

**Pull request merged**